thoth/user_api/api_v1.py

```diff
@@ -268,9 +268,9 @@ def get_provenance_python_log(analysis_id: str) -> typing.Tuple[typing.Dict[str,
     return _get_log("provenance-check", analysis_id, namespace=Configuration.THOTH_BACKEND_NAMESPACE)
 
 
-def get_provenance_python_status(analysis_id: str):
+def get_provenance_python_status(analysis_id: str) -> typing.Tuple[typing.Dict[str, typing.Any], int]:
     """Get status of a provenance check."""
-    return _get_workflow_status(locals(), "provenance-checker-", Configuration.THOTH_BACKEND_NAMESPACE)
+    return _get_status("provenance-checker-", analysis_id, namespace=Configuration.THOTH_BACKEND_NAMESPACE)
```

Comment:
should it be `"provenance-check"` or `"provenance-checker"` instead of `"provenance-checker-"`?

Reply:
it's `provenance-check`